cms/envs/common.py

```diff
@@ -121,6 +121,9 @@
     # Methods to derive settings
     _make_mako_template_dirs,
     _make_locale_paths,
+
+    # Password Validator Settings
+    AUTH_PASSWORD_VALIDATORS
```

Comment:
[log] This introduces a change making the CMS `MinimumLengthValidator` default to 8 instead of 2, which is desired behavior if I understand correctly!